.github/ISSUE_TEMPLATE/bug_report_form.yml

```diff
@@ -0,0 +1,203 @@
+      placeholder: docker-compose version 1.28.6, build 5db8d86f
+    validations:
+      required: false
+  - type: input
+    id: uname-a
+    attributes:
+      label: The output of `uname -a`
+      description: Please paste the output of `uname -a` here.
+      placeholder: Linux swift 5.11.0-31-generic \#33-Ubuntu SMP Wed Aug 11 13:19:04 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
+    validations:
+      required: true
+  - type: textarea
+    id: important-environment-variables
+    attributes:
+      label: Important environment variables
+      description: If there are environment variables important to this issue, please post them here. If we ask for the values of variables, make sure to put them in here too.
+      render: bash
+  - type: textarea
+    id: relevant-log-output
+    attributes:
+      label: Relevant log output
+      description: Show us relevant log output here. You can enable debug output by setting `DMS_DEBUG` to `1`.
+      render: shell
+  - type: textarea
+    id: other-relevant-information
+    attributes:
+      label: Other relevant information
+      description: If there is more, you can tell us here. Make sure to use Markdown formatting here.
+      render: Markdown
+  - type: dropdown
+    id: experience
+    attributes:
+      label: What level of experience do you have with Docker and mail servers?
+      description: You are not obliged to answer this question. But answering it helps us better understand where possible causes of errors lie. Less experienced users are more likely do make "obvious" mistakes, which is fine, and by answering this question, we know to look for them as well. We encourage everyone to participate, regardless of their level of experience.
+      options:
+        - I'm very much a beginner
+        - I'm a novice
+        - I know containerization but have less experience with mail
+        - I have experience with mail but less experience with containerization
+        - I know both pretty well and I'm experienced
+        - I'm an expert
+        - Trust me, I'm a (computer) engineer
+    validations:
+      required: false
+  - type: checkboxes
+    id: terms-code-of-conduct
+    attributes:
+      label: Code of conduct
+      description: By submitting this issue, you agree to follow [our code of conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md).
+      options:
+        - label: I read this project's Code of Conduct and I agree
+          required: true
+        - label: I read the README and the documentation but could not find a solution
+          required: true
+  - type: checkboxes
+    id: terms-miscellaneous
+    attributes:
+      label: Miscellaneous
+      description: Please read these carefully and check the appropriate boxes.
```

Comment:
Not required, but at a later point if these turn out to be common issues that get explained to users during troubleshooting, perhaps have both the ports and SSL troubleshooting as doc pages and link to them here.

May be relevant early on so they're not cursing by the time they get to the end :sweat_smile:

Reply:
I moved this section to the beginning.